plasmapy/physics/tests/test_transport.py

```diff
@@ -1388,7 +1460,10 @@ def test_ion_thermal_conductivity_by_model(self, model, expected):
                           5.08966116e-06]}.items())
     def test_dictionary(self, key, expected):
         calculated = self.all_variables[key]
-        assert np.allclose(expected, calculated.si.value)
+        testTrue = np.allclose(expected, calculated.si.value)
+        errStr = (f"Expected values of {key} are {expected} and not"
+                  f"{calculated.si.values}.")
```

Comment:
If you take a look at the [test results](https://travis-ci.org/PlasmaPy/PlasmaPy/builds/339017903#L1256), it seems this line is killing all the failing tests due to a typo:
```
.values -> .value
```
If you fix that, we should be good to go with merging this PR!